app/assets/stylesheets/bootstrap.css

```diff
@@ -6118,7 +6117,6 @@ button.close {
   font-weight: normal;
   line-height: 1.42857143;
   text-align: left;
-  text-align: start;
```

Comment:
Since you're taking an override approach, I would not expect anything to change in this file. Is there a reason these changes can't be overridden in bootstrap-rtl.scss?

Reply:
I deleted this line in the original file because I thought it is a duplication, Isn't it a duplication?

Reply:
This was in the original, which we didn't write. I suspect they added it as a progressive enhancement to help with RTL support. Regardless, I just wouldn't change this file.